style2.css

```diff
@@ -0,0 +1,145 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  max-width: 375px;
+  height: 50px;
+}
+
+.welcome {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: semi bold;
+  font-size: 20px;
+  line-height: 28px;
+  margin-left: 35px;
+  display: inline-block;
+}
+
+.union {
+  width: 18px;
+  height: 12px;
+  display: inline-block;
+  margin-left: 230px;
+}
+
+.flex {
+  max-width: 375px;
+  flex-wrap: wrap;
+}
+
+.hi {
+  font-weight: bold;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 45px;
+  line-height: 52px;
+  color: #172b4d;
+  position: absolute;
+  top: 135px;
+  left: 35px;
+}
+
+.first-name {
+  font-weight: bold;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 45px;
+  line-height: 52px;
+  color: #172b4d;
+  position: absolute;
+  top: 190px;
+  left: 95px;
+}
+
+.job {
+  font-weight: bold;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 45px;
+  line-height: 52px;
+  color: #36b37e;
+  position: absolute;
+  left: 35px;
+  top: 250px;
+}
+
+.job-info {
+  font-weight: bold;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 45px;
+  line-height: 52px;
+  color: #36b37e;
+  position: absolute;
+  left: 95px;
+  top: 310px;
+}
+
+.information-box {
+  width: 550px;
+  height: 400px;
+  align-items: center;
+  position: absolute;
+  top: 400px;
+}
+
+.info {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 0.5;
+  color: #172b4d;
+  margin-left: 35px;
+}
+
+.info2 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 0.5;
+  color: #172b4d;
+  margin-left: 35px;
+}
+
+.info3 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 0.5;
+  color: #172b4d;
+  margin-left: 35px;
+}
+
+.info4 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 0.5;
+  color: #172b4d;
+  margin-left: 35px;
+}
+
+.info5 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 0.5;
+  color: #172b4d;
+  margin-left: 35px;
+}
+
+.info6 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
```

Comment:
- [ ] Kindly remove this lines from your CSS since you are repeating them too much and this is not a good practice for DRY principle. You don't need to declare font-style as normal since it's default. Also when you want every item have the same feature then you can use this selector `*{}` at the beginning.